spt/features/playerio.cpp

```diff
@@ -412,6 +419,11 @@ void __fastcall PlayerIOFeature::HOOKED_CreateMove_Func(void* thisptr,
 
 	ORIG_CreateMove(thisptr, edx, sequence_number, input_sample_frametime, active);
 
+#if defined(SSDK2007)
+	auto cmd = reinterpret_cast<CUserCmd*>(pCmd);
+	spt_ihud.SetInputInfo(cmd->buttons, Vector(cmd->sidemove, cmd->forwardmove, cmd->upmove));
+#endif
+
```

Comment:
This would be better if you added a new signal into spt/utils/signals.hpp/cpp, for example `Gallant::Signal1<CUserCmd*> CreateMove;` then you could connect to it inside ihud.

Reply:
For example see SV_ActivateServerSignal in generic.cpp